backend/de.metas.business/src/main/java/de/metas/bpartner/composite/BPartnerComposite.java

```diff
@@ -335,4 +336,15 @@ public Stream<BPartnerLocationId> streamBPartnerLocationIds()
 	{
 		return this.getLocations().stream().map(BPartnerLocation::getId);
 	}
+
+	@Nullable
+	public String getOrgCode(@NonNull final Function<@NonNull OrgId,@NonNull String> orgId2String)
```

Comment:
hey @teosarca when you have time, please let us know what you think about this

Reply:
@metas-ts  @av-ps feels pretty awkward TBH.

loud thinking: 
wondering why not having `Optional<String> getOrgCode()`, so we would use it as
```java
bpartnerComposite.getOrgCode().map(orgDAO::getOrgCode).orElse(null)
```

yeah... a bit longer but IMHO much more intention revealing.

The `getOrgCode(@NonNull final Function<@NonNull OrgId,@NonNull String> orgId2String)` would be intention revealing if we would call it "mapOrgCodeIfNotNull`.